setup.py

```diff
@@ -1,3 +1,3 @@
 from setuptools import setup
 
-setup()
+setup(extras_require={"reports": ["pydantic>=2.0.0"]})
```

Comment:
Is this the right place for extras @kptkin ?

Reply:
`pyproject.toml` => [project.optional-dependencies]

Reply:
like @vanpelt said, you can see other extras for examples.